nixos/modules/system/boot/systemd/initrd.nix

```diff
@@ -584,5 +584,8 @@ in {
         serviceConfig.Type = "oneshot";
       };
     };
+
+    # disable emergency mode if the user cannot interact with it
+    systemd.enableEmergencyMode = lib.mkIf (cfg.emergencyAccess == false) false;
```

Comment:
`nixos/modules/system/boot/systemd/initrd.nix` says this:

```nix
    emergencyAccess = mkOption {
      type = with types; oneOf [ bool (nullOr (passwdEntry str)) ];
```

Should this be testing for null too? Actually, what does `null` even mean? Based on the type, it looks like that's allowed as well.